src/constants/todo.general.js

```diff
@@ -0,0 +1,7 @@
+const EMPTY_ARRAY = [];
```

Comment:
***** VERY IMPORTANT *****

File convention followed here :

If we have a single file containing miscellaneous helpers, then we create it at the folder level itself, example:

folder/name.helpers.js : Named exports
 
If we have a single file and it has helpers related to a particular context, then we create a helpers directory for it, example:

folder/helpers/name.{context}.js : Default export 
 
If we have multiple helper filters, then a helpers directory is inevitable

Same goes for constants.